sorc/ncep_post.fd/ALLOCATE_ALL.f

```diff
@@ -70,6 +72,48 @@ SUBROUTINE ALLOCATE_ALL()
       allocate(train(im,jsta_2l:jend_2u,lm))
       allocate(tcucn(im,jsta_2l:jend_2u,lm))
       allocate(EL_PBL(im,jsta_2l:jend_2u,lm))
+
+!Initialization
+      do l=1,lm
+        do j=jsta_2l,jend_2u
+          do i=1,lm
+            u(i,j,l)=0.
+            v(i,j,l)=0.
+            t(i,j,l)=spval
+            q(i,j,l)=spval
+            uh(i,j,l)=spval
+            vh(i,j,l)=spval
+            wh(i,j,l)=spval
+            pmid(i,j,l)=spval
+            pmidv(i,j,l)=spval
+            zmid(i,j,l)=spval
+            q2(i,j,l)=spval
+            omga(i,j,l)=spval
+            T_ADJ(i,j,l)=spval 
+            ttnd(i,j,l)=spval 
+            rswtt(i,j,l)=spval 
+            rlwtt(i,j,l)=spval 
+            exch_h(i,j,l)=spval 
+            train(i,j,l)=spval 
+            tcucn(i,j,l)=spval 
+            EL_PBL(i,j,l)=spval 
+            u(i,j,l)=spval 
+            u(i,j,l)=spval 
+            u(i,j,l)=spval 
```

Comment:
there are 4 Us being initialized here

Reply:
@HuiyaChuang-NOAA Thanks for catching it.